app/models/conversion_host.rb

```diff
@@ -241,20 +241,27 @@ def resource_supports_conversion_host
   end
 
   # Find the credentials for the associated resource. By default it will
-  # look for a v2v auth type. If that is not found, it will look for the
-  # authentication associated with the resource using ssh_keypair or default,
-  # in that order, as the authtype.
+  # look for a v2v auth type if no argument is passed in. If that is not found,
+  # it will look for the first associated authentication. If one still isn't
+  # found, then it will look for the authentication associated with the resource
+  # using the 'ssh_keypair' auth type, and finally 'default'.
   #
-  def find_credentials(msg = nil)
-    authentication = authentication_type('v2v') ||
-      resource.authentication_type('ssh_keypair') ||
-      resource.authentication_type('default')
+  def find_credentials(auth_type = nil)
```

Comment:
Can you make this `auth_type = 'v2v' so you can remove one of the conditions down [here](https://github.com/ManageIQ/manageiq/pull/18724/files#diff-02f7c651ed336a288d5c05feffcf761bR250)

Reply:
I was originally going to do that, but if some code explicitly passed in a different auth type, I wanted it to still fall back to checking for v2v.

Reply:
If they're explicitly asking for a specific auth type then I wouldn't worry about falling back

Reply:
Ok, done!